build-logic/project-plugins/src/main/kotlin/com.hedera.hashgraph.sdk.conventions.gradle.kts

```diff
@@ -33,11 +30,17 @@ configurations.getByName("mainRuntimeClasspath") {
 
 gitProperties { keys = listOf("git.build.version", "git.commit.id", "git.commit.id.abbrev") }
 
-testlogger {
-    theme = ThemeType.MOCHA
-    slowThreshold = 10000
-    showStandardStreams = true
-    showPassedStandardStreams = false
-    showSkippedStandardStreams = false
-    showFailedStandardStreams = true
+// === Remove the following once 'test' tasks are allowed to run in parallel ===
+val allProjects = rootProject.subprojects.map { it.name }.filter {
+    it !in listOf("swirlds", "swirlds-sign-tool") // these are 'application' projects
+}.sorted()
+val myIndex = allProjects.indexOf(name)
+if (myIndex > 0) {
+    val predecessorProject = allProjects[myIndex - 1]
+    tasks.test {
+        mustRunAfter(":$predecessorProject:test")
+        mustRunAfter(":$predecessorProject:hammerTest")
+        mustRunAfter(":$predecessorProject:performanceTest")
+    }
```

Comment:
@hendrikebbers can we create a follow up issue to look at the problematic tests. There are probably just a few which are too timing sensitive. I suspect thing like `SECOND.sleep(1)` [here](https://github.com/hashgraph/hedera-services/blob/058dc9b607a657abee927a573d8feaff3cb40484/platform-sdk/swirlds-unit-tests/common/swirlds-common-test/src/test/java/com/swirlds/common/test/threading/StoppableThreadTests.java#L473) to be responsible.

Here are the failing tests (might be more):
- https://scans.gradle.com/s/d2guxjyk22hkq/tests/overview?outcome=FAILED
- https://scans.gradle.com/s/lvjijevluzfsy/tests/overview?outcome=FAILED
- https://scans.gradle.com/s/xfdnlxtdr7e5y/tests/overview?outcome=FAILED